### PR TITLE
Emphasized columns styling

### LINKED ISSUE #68: implement variants of the columns block
We need a variant of the columns block with slightly more emphasis.

Image

For instance:
- https://www.delta.com/us/en/skymiles/how-to-earn-miles/exception-fares

### PR DESCRIPTION
**Pull request by asthabh23**
Fix #68 

Test URLs:
- Before: https://main--delta--hlxsites.hlx.page/drafts/asthabharga/us/en/skymiles/how-to-earn-miles/exception-fares
- After: https://issue-68--delta--hlxsites.hlx.page/drafts/asthabharga/us/en/skymiles/how-to-earn-miles/exception-fares

### CODE REVIEW

**Comment by asthabh23** [comment]
Note: I have kept the width of two parts of column equal on Franklin site (between `768px` and `992px`) keeping it better visually, in comparison to how it is on original site. Sharing attachments below:

Franklin implementation:
Screenshot 2023-03-17 at 3 00 33 PM

Original site:
Screenshot 2023-03-17 at 3 00 50 PM

**Review by ramboz — APPROVED** [review]
Looks good to me

File: us/en/skymiles/blocks/columns/columns.css [hunk 1]
```diff
@@ -25,6 +25,39 @@
...
+    margin-left: 0;
+    text-align: left;
+}
+
+.columns.emphasized > div > div > p {
+    color: var(--color-delta-blue);
+    font-size: var(--heading-font-size-m);
+}
+
+.columns.emphasized > div > div > p:first-child {
```
Comment (askayastha22) [inline us/en/skymiles/blocks/columns/columns.css:55]: The selectors seem complex here. @davidnuescheler previously recommended me to keep selectors simple by absorbing some of the complexity to JS.

**Review by askayastha22 — COMMENTED** [review]
There are a few complex CSS selectors (first-child, last-child, etc.), which we may want to simplify based on the best practices recommended by @davidnuescheler

File: us/en/skymiles/blocks/columns/columns.css (hunk 1 above)
Comment (ramboz) [reply us/en/skymiles/blocks/columns/columns.css:55]: I would say it's fine for now as this is a 1st iteration on this block variant. As we go through the rest of the pages we'll see how to refine this, and maybe in the end we just move this back to content formatting in Word and just keep plain layout stuff in the CSS

Comment (asthabh23) [reply us/en/skymiles/blocks/columns/columns.css:55]: Just to add, we try to focus more on including JS when we have a bunch of say <p> tags where you we need to style few of them. Considering that this column was pretty straight forward with just two divs within it, I tried to keep it simple with basic styling .But ofcourse if we see more variants for such block in other pages and DOM manipulation seems to be the need, we can surely go for it.

**Pull request merged**